sensus/sensus/code.py

```diff
@@ -45,6 +108,9 @@ def update_mini_batch(self, mini_batch, eta): # eta is just a constant that dete
             grad_w = [nw+dnw for nw, dnw in zip(grad_w, delta_grad_w)]
```

Comment:
Super tiny detail but I think good style calls for spaces before and after these signs. Also in the line above but I cannot comment on that somehow.
```suggestion
            grad_w = [nw + dnw for nw, dnw in zip(grad_w, delta_grad_w)]
```